tools/api-markdown-documenter/src/api-item-transforms/configuration/index.ts

```diff
@@ -16,8 +18,8 @@ export {
 	type HierarchyBoundaries,
 } from "./DocumentationSuiteOptions.js";
 export {
-	type ApiItemTransformationOptions,
-	getApiItemTransformationOptionsWithDefaults,
+	type ApiItemTransformations,
+	getApiItemTransformationsWithDefaults as getApiItemTransformationOptionsWithDefaults,
```

Comment:
Keeping the alias is intended?

Reply:
Ugh. Nope. Good catch. Thanks!

Reply:
Actually, this didn't need to be exported at all! Removed!